langchain/vectorstores/redis.py

```diff
@@ -461,19 +464,24 @@ def from_texts(
 
     @staticmethod
     def delete(
-        keys: List[str],
+        ids: Optional[List[str]] = None,
```

Comment:
lets just change this, since its an arg and idk how well used i dont think we need to worry about backwards compat